### PR TITLE
Added required TOs and basic tests for service schedules API.

### PR DESCRIPTION
**Pull request by douradofrazer**

### CODE REVIEW

File: ovs-transfer-obj/src/main/java/org/dcsa/ovs/transferobjects/enums/DimensionUnit.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package org.dcsa.ovs.transferobjects.enums;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public enum DimensionUnit {
+  MTR("Meter"),
+  FOT("Foor");
```
[the remark this answers is not in the document]
Comment (douradofrazer) [reply ovs-transfer-obj/src/main/java/org/dcsa/ovs/transferobjects/enums/DimensionUnit.java]: Fixed.

File: ovs-service/src/main/java/org/dcsa/ovs/controller/OVSRestController.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  }
+
+  // TODO: Move to a handler
+  @ExceptionHandler(ConstraintViolationException.class)
+  public ResponseEntity<RequestFailureTO> badRequest(
+      HttpServletRequest httpServletRequest, ConstraintViolationException cvex) {
+
+    String exceptionMessage = null;
+
+    if (Objects.nonNull(cvex.getConstraintViolations())) {
```
Comment (FinnAtAsseco) [inline ovs-service/src/main/java/org/dcsa/ovs/controller/OVSRestController.java:51]: Please just use == null. Objects.nonNull is not meant to be used in this way.

File: ovs-service/src/main/java/org/dcsa/ovs/controller/OVSRestController.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+      exceptionMessage =
+          cvex.getConstraintViolations().stream()
+              .filter(Objects::nonNull)
```
Comment (FinnAtAsseco) [inline ovs-service/src/main/java/org/dcsa/ovs/controller/OVSRestController.java:54]: Yes, that's what Objects.nonNull is meant for.

**Review by pimg — APPROVED** [review]

File: ovs-service/src/main/java/org/dcsa/ovs/controller/OVSRestController.java (hunk 2 above)
Comment (douradofrazer) [reply ovs-service/src/main/java/org/dcsa/ovs/controller/OVSRestController.java:51]: It is temporary code, for now, to comply with the error format, I will change it when I extract it out later.

**Pull request merged**